bundles/org.openhab.binding.ventaair/README.md

```diff
@@ -0,0 +1,119 @@
+# VentaAir Binding
+
+This binding is for air humidifiers from Venta Air.
+Thankfully the vendor allows for communicating within the local network without needing any internet access or accounts.
+This is even stated in the official manual.
+Hence this binding communicates locally with the humidifier and is able to read out the current measurements and settings, as well as changing the settings.
+
+## Supported Things
+
+It currently supports the LW60-T device as well as a generic `ThingType` for other models.
+For now the generic `ThingType` only adds the "boost" channel, but in the status reply from the device there is more which could be added in the future by someone who owns a different device.
+
+## Discovery
+
+This binding supports an automatic discovery for humidifiers that are connected to the local network and which are in the same broadcast domain.
+To do so, the binding listens to UDP port 48000 for data and creates `DiscoveryResult`s based on the received data from the device.
+This comes in handy for getting the MAC address for the device for example.
+Once the `DiscoveryResult` is added as a `Thing`, a connection to the device will be created and it will beep, showing a confirmation screen that the device "openHAB" would like to get access.
+After confirming this request, the use can link its `Item`s to receive data or control the device.
+
+## Thing Configuration
+
+There are three mandatory configuration parameters for a thing: `ipAddress`, `macAddress` and `deviceType`.
+Whereas the first two are self-exploring, the later one defines the type of the device. It is an integer value and its best to use the automatic discovery to obtain it from the device.
+The optional parameter `pollingTime` is the time interval in seconds in which the data should be polled from the device, default is 10 seconds.
+The `hash` parameter is also optional. It is a negative integer value and it is used by the device to identify a connection to a client, like the App from the vendor for example.
```

Comment:
It may be better to have this information in a table format, check the other bindings on how they do this. You have done it for the channels already which is good. Also each sentence should be on its own line.